### DISCUSSION TITLE
Cold wallet reward transfer

### DISCUSSION THREAD

**Discussion by Ehtxt** [guydavis/machinaris#495] [Q&A] [answered]
So i made cold wallets for all the chia forks and i have a question
1. Does the reward go to my cold wallet directly which i inserted in my wallets section?
Now according to the guides i should include a puzzle hash in config.yaml file but i cannot find where to include it,
and also I don't want to use pools and i am solo mining chia, so it would be really helpful if someone can guide me to setup cold wallet in machinaris to which the rewards get transferred automatically.
I have attached a snap from config.yaml, should i change the target address?
Thank you!!!
![11111](https://user-images.githubusercontent.com/22908335/145728848-ee309bc0-6525-4d7c-9853-4802696e78f8.JPG)

**Comment by guydavis** [comment, marked as answer]
Hi, please see the [wiki](https://github.com/guydavis/machinaris/wiki/Wallets#cold-wallet).
